packages/vuetify/src/components/VDatePicker/VDatePickerDateTable.ts

```diff
@@ -47,7 +54,142 @@ export default mixins(
     },
   },
 
+  watch: {
+    tableDate () {
+      setTimeout(() => {
+        this.dateButtons = Array.from(this.$el.querySelectorAll('td'))
+        if (this.focusedDateIndex > this.dateButtons.length - 1 || this.checkIfElemEmpty(this.focusedDateIndex)) {
+          this.setStartIndex()
+        }
+      }, 500)
```

Comment:
where does this 500 come from?

Reply:
Replaced timeout logic with `afterLeave` transtion hook